[K9VULN-5572] Add section for why libraries might be missing (#29723)
DataDog/documentation: content/en/security/code_security/troubleshooting/_index.md

If any of these conditions apply to your repository, and you cannot update your repository to account for these constraints, [set up the analysis in a CI pipeline][19] to run SCA and upload results to Datadog.

### Missing libraries

> To ensure data quality, Datadog applies validation rules during SBOM processing. Libraries that meet any of the following criteria will be excluded:

Reviewer: ```suggestion
To ensure data quality, Datadog applies validation rules during SBOM processing. Libraries that meet any of the following criteria are excluded:
```

- **Missing version**: The library does not specify a version.
- **Non-ASCII name**: The library name contains characters outside the ASCII character set.
- **Empty purl**: The package URL (purl) field is missing or blank.
- **Invalid purl**: The package URL is present but not in a valid purl format.

> - **Unsupported language**: The library is associated with a programming language that Datadog does not currently support.

Reviewer: ```suggestion
- **Unsupported language**: The library is associated with a programming language that Datadog does not support.
```
